qutebrowser/browser/network/networkmanager.py

```diff
@@ -241,12 +241,21 @@ def clear_rejected_ssl_errors(self, url):
     @pyqtSlot('QNetworkReply', 'QAuthenticator')
     def on_authentication_required(self, reply, authenticator):
         """Called when a website needs authentication."""
-        answer = self._ask("Username ({}):".format(authenticator.realm()),
+        import netrc
+        net = netrc.netrc()
+        authenticators = net.authenticators(reply.url().host())
+        user, password = None, None
+
+        if authenticators:
+            (user, account, password) = authenticators
+        else:
+            answer = self._ask("Username ({}):".format(authenticator.realm()),
                            mode=usertypes.PromptMode.user_pwd,
                            owner=reply)
```

Comment:
As pointed out by `pep8`, those should be indented below the `"`, i.e. after the `(`.